docs/FAQ.md

```diff
@@ -537,13 +537,23 @@ commits associated with it.
 
 ### How do I integrate Jujutsu with Gerrit?
 
-At the moment you'll need a script, which adds the required fields for Gerrit
-like the `Change-Id` footer. Then `jj` can invoke it via an `$EDITOR` override
-in an aliased command. Here's an [example][gerrit-integration] from a
-contributor (look for the `jj signoff` alias).
+Add this to your configuration:
```

Comment:
nit: I think this should be explicit about what this is for. Something like "Add this to your configuration to automatically add Change-Id footers to commit messages:"